src/app/legacy/containers/Header/index.jsx

```diff
@@ -76,7 +79,8 @@ const HeaderContainer = ({
     </SkipLink>
   );
 
-  const shouldRenderScriptSwitch = scriptLink && renderScriptSwitch;
+  const shouldRenderScriptSwitch =
+    scriptLink && isOptimoId && renderScriptSwitch;
```

Comment:
This will probably prevent the script switch button appearing on non-Article pages like the homepage.

It'll likely need extended to cater for the individual Optimo asset condition, so potentially something like:

```
const shouldRenderScriptSwitch = (scriptLink && renderScriptSwitch) || (scriptLink && renderScriptSwitch && isOptimoId);
```

Reply:
Actually for the logic to work I think it will need narrowed to Uzbek.